apps/frontend/src/modules/courses/slices/courses.ts

```diff
@@ -1,11 +1,21 @@
-import { getAll, getById, getRecommended } from "./actions.js";
+import {
+	deleteById,
+	getAll,
+	getAllByFilter,
+	getById,
+	getRecommended,
+	update,
+} from "./actions.js";
 import { actions } from "./courses.slice.js";
 
 const allActions = {
 	...actions,
+	deleteById,
 	getAll,
+	getAllByFilter,
```

Comment:
```suggestion
	deleteById,
	getAll,
```
getAll should be the same `getAllByFilter` but with nullises params. so you need to keep only one action